composer.json

```diff
@@ -27,6 +27,7 @@
     "ext-curl": "*",
     "ext-simplexml": "*",
     "ext-mbstring": "*",
+    "ext-json": "*",
```

Comment:
yes, to my knowledge it is needed for `json_encode()` and `json_decode()`. As it is not 100% ensured that is installed on the system I added this line. With php8 it can be removed again ;-)

Reply:
https://www.php.net/manual/en/json.installation.php